### PR TITLE
Publish Crowdin monthly October 2020

### PR DESCRIPTION
**Pull request by Julievrz**
Publish Crowdin monthly - October 2020.

### CODE REVIEW

File: news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+subtitle: "Where you'll learn that the last 1.7.7 translations are available"
+date:  2020-11-09 12:00:00
+authors: [JulieVarisellaz]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! 
+This report will tell you how the translations of the software evolved in October!
```
Comment (Julievrz) [inline news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md]:
```suggestion
This report will tell you how the translations of the software evolved in October.
```

File: news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+authors: [JulieVarisellaz]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! 
+This report will tell you how the translations of the software evolved in October.
+
+[Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
```
Comment (Julievrz) [inline news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md]:
```suggestion
![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
```

**Comment by Julievrz** [comment]
It seems that there is an issue with the display of my images... I don't why it sometimes displays well and sometimes not... 😅

**Review by atomiix — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by ttoine** [comment]
I see that the publication is Monday.
@Julievrz feel free to merge it the right day, then

File: news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+This is less than in September, but this is not surprising as new translations have been added this month.
+ 
+## Languages with the best evolution
+ 
+In comparison with October 2020, the following languages had the best progress thanks to the translation community:
+ 
+* **Azerbaijani** (+9% to reach 37% translated)
+* **French, Belgium** (+8% to reach 8% translated)
+* **English** (+5% to reach 100% translated)
```
Comment (Julievrz) [inline news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md]:
```suggestion
* **Esperanto** (+2% to reach 59% translated)
```

File: news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+Remember, you can see who's been contributing to our translation project thanks to the [Translators page](http://translators.prestashop.com/).
+ 
+ 
+## Complete translations
+ 
+#### Fully translated languages
+ 
+At the end of October 2020, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **6 languages**:
+ 
+| Czech | English | Spanish | Portuguese, Brazil |
```
Comment (Julievrz) [inline news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md]:
```suggestion
| **Czech** | **English** | Spanish | Portuguese, Brazil |
```

File: news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+This is less than in September, but this is not surprising as new translations have been added this month.
+ 
+## Languages with the best evolution
+ 
+In comparison with October 2020, the following languages had the best progress thanks to the translation community:
+ 
+* **Azerbaijani** (+9% to reach 37% translated)
+* **French, Belgium** (+8% to reach 8% translated)
+* **Esperanto** (+2% to reach 59% translated)
```
Comment (Julievrz) [inline news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md]:
```suggestion
* **Esperanto** (+6% to reach 19% approved)
```

File: news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+At the end of October 2020, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **6 languages**:
+ 
+| **Czech** | **English** | Spanish | Portuguese, Brazil |
+| Slovak | Swedish | 
+
+This is less than in September, but this is not surprising as new translations have been added this month.
+ 
+## Languages with the best evolution
+ 
+In comparison with October 2020, the following languages had the best progress thanks to the translation community:
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md]:
```suggestion
In comparison with September 2020, the following languages had the best progress thanks to the translation community:
```

**Comment by Julievrz** [comment]
> I see that the publication is Monday.
> @Julievrz feel free to merge it the right day, then

Hello @ttoine , 

Since the article is ready a bit earlier, would it be possible to publish it today? 

Otherwise, Louise will publish it on Monday.

**Comment by ttoine** [comment]
why not ?

File: news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – October 2020 edition"
+subtitle: "Where you'll learn that the last 1.7.7 translations are available"
+date:  2020-11-09 12:00:00
```
Comment (Julievrz) [inline news/_posts/translation/2020-11-09-do-you-speak-prestashop-oct-2020.md]:
```suggestion
date:  2020-11-06 12:00:00
```

**Pull request merged**